roles/ci_gen_kustomize_values/molecule/default/converge.yml

```diff
@@ -79,6 +79,52 @@
           examples/va/nfv/sriov/edpm/values.yaml
         cifmw_ci_gen_kustomize_values_userdata: >-
           {{ cifmw_reproducer_user_edpm_values }}
-        cifmw_ci_gen_kustomize_values_dest_filename: 'edpm-values.yml'
       ansible.builtin.include_role:
         name: ci_gen_kustomize_values
+
+    - name: Get generated files status
+      register: stat_values
+      ansible.builtin.stat:
+        get_attributes: false
+        get_checksum: false
+        get_mime: false
+        path: >-
+          {{
+            (ansible_user_dir, 'ci-framework-data',
+             'artifacts', 'ci_gen_kustomize_values',
+             item, 'values.yaml') |
+             path_join }}
+      loop:
+        - network-values
+        - edpm-values
+
+    - name: Assert generated values.yaml exists
+      ansible.builtin.assert:
+        that:
+          - item.stat.exists
+      loop: "{{ stat_values.results }}"
+      loop_control:
+        label: "{{ item.item }}"
+
+    - name: Copy generated values to correct location
+      ansible.builtin.copy:
+        remote_src: true
+        src: "{{ artifacts }}/ci_gen_kustomize_values/{{ item.key }}/values.yaml"
+        dest: "{{ cifmw_ci_gen_kustomize_values_architecure_repo }}/{{ item.value }}"
+      loop:
+        - key: 'network-values'
+          value: 'examples/va/nfv/sriov/values.yaml'
+        - key: 'edpm-values'
+          value: 'examples/va/nfv/sriov/edpm/values.yaml'
+
+    - name: Ensure kustomize is able to build
```

Comment:
I like having this extra test so overall I like this patch.

One thought: if you are going to pull from the architecture repo and then modify that file but the source at the architecture repo broke for some reason, then this test might turn up false positives. One way to prevent that is to have a sanity test which pulls the CR from the architecture repos and runs kustomize on it before anything changes. That would tell you that if this new job fails that it wasn't due to a problem in a proposed patch to ci-framework but a problem in the architecture repo.

Reply:
so running twice? Might be a thing indeed. BTW, maybe a job against the `architecture` repository should do the `run kustomize` ? To ensure no breaking change may slip in?

Reply:
Yeah, I'd like it if the architecture repository had a job but that repository has no automated jobs right now. Thus, adding an extra task or two to this PR to run kustomize first will prevent this problem for you now.

Reply:
just added that as a look, and we'll generate "vanilla-{{ file }}" we may compare with the ones we generate later if we want.

Reply:
Seems to work as expected:
https://logserver.rdoproject.org/73/1073/f3f345dd3fbc860319094cbd7c89c97955b3513a/github-check/cifmw-molecule-ci_gen_kustomize_values/0c2016f/ci-framework-data/artifacts/

We have the 4 wanted files.

Reply:
+2